tests/components/auth/test_init.py

```diff
@@ -76,6 +77,14 @@ def test_credential_store_expiration():
             'api_password': 'bla'
         }
     })
+
+    user = hass_access_token.refresh_token.user
+    credential = Credentials(auth_provider_type='homeassistant',
+                             auth_provider_id='hass-test-id',
+                             data={}, id='test-id')
+    user.credentials.append(credential)
```

Comment:
We should move this to tests/components/conftest.py::hass_access_token()

Reply:
This is breaking other tests